sdk/spring-sdk/src/main/scala/kalix/springsdk/impl/KalixClient.scala

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright 2021 Lightbend Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package kalix.springsdk.impl
+
+import akka.annotation.ApiMayChange
+import kalix.javasdk.{ DeferredCall, Metadata }
+import org.springframework.http.{ HttpHeaders, MediaType }
+import org.springframework.web.reactive.function.client.WebClient
+
+import java.util.concurrent.CompletionStage
+import scala.concurrent.{ Future, Promise }
+import scala.jdk.FutureConverters._
+import scala.concurrent.ExecutionContext.Implicits.global
+
+trait KalixClient {
```

Comment:
I think this should move to `kalix.springsdk` package. 

It's an API that we are providing for users delight. 😄

Reply:
absolutely, will update. thanks Renato.